src/styles.scss

```diff
@@ -11,8 +11,8 @@
 body {
```

Comment:
in global styles should be no styles that can affect other elements

Reply:
i remained any stiles for body like color, bgc